frontend/src/components/collective/CollectiveExpenseItem.js

```diff
@@ -39,6 +39,13 @@ const CollectiveExpenseItem = ({
 
   let src, srcSet, href, receiptMessage, receiptHelpTip;
 
+  let submittedByName = user && user.name;
+
+  // show email only if the user can view receipt
+  if (canViewReceipt) {
```

Comment:
if canViewReceipt or if canEdit?
Because if we want to allow people to cover the expense in the future, everyone should be able to see it.